idea_town/frontend/static-src/app/templates/scroll-header-view.js

```diff
@@ -8,7 +8,12 @@ export default `
         </header>
         {{^scrolled}}
           <div id="avatar-wrapper">
-            <span class="default-avatar" data-hook="logout"></span>
+            {{#avatar}}
+              <img class="avatar" src="{{avatar}}" width="41" height="41" data-hook="logout">
```

Comment:
Hmm, that could get vaguely complicated. We can't hardcode a literal `/static/`, because that's actually a configurable path we might change if/when we go to a CDN / S3 per #141